decidim-sortitions/app/models/decidim/sortitions/sortition.rb

```diff
@@ -2,7 +2,7 @@
 
 module Decidim
   module Sortitions
-    # Model that encapsulates the parameters of a sortion
+    # Model that encapsulates the parameters of a sortition
```

Comment:
I don't recognize this term, but I'm treating it as a term-of-art and just making it match the module.